### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by gdvl777**
#Hello Microverse
New Pull Request because of creation of linters, index, style and test files.

### CODE REVIEW

File: .github/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
```
Comment (topeogunleye) [inline .github/linters.yml]: - [x] Kindly add linters to your project. In the root project, you need to have a folder named `.github`. Inside this one, you need another folder named `workflows`. And inside the workflows folder, you need a file named `linters.yml`. Inside this file, you need to have the linters. The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css);
Here is a video demo I created demonstrating  the above:
>
https://user-images.githubusercontent.com/52934128/164221056-46f41b39-cb1e-4fd9-a7f9-dd47bf835439.mp4
- [ ] Kindly ensure your linter results pass * AKA: Linter should report NO errors.

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
-# github-flow
\ No newline at end of file
+# github-flow
+
+Brief description of the project.
```
Comment (topeogunleye) [inline README.md]: - [x] In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
  - Kindly add all sections from the Microverse README.
  - The only optional section from the README file is `faq`.

| professional Readme  | Your Readme that is not professional |
| ------------- | ------------- |
|  Recording 2023-02-14 at 12 52 20 (5)	  | image  |

File: .github/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules/
```
Comment (topeogunleye) [inline .github/.gitignore:1]: - [x] Kindly ensure that your ignore is not in your `.github` folder but instead in your `root` folder. The project root is the folder which is the parent for all the project sources

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi  @gdvl777, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._
 - [x] Please kindly add a descriptive PR title and description as seen in the demo below:
pull_request_desc

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gdvl777 — COMMENTED** [review]
The changes requested have been solved.

**Comment by gdvl777** [comment]
> Hi @gdvl777,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> Try-Again [ Try-Again ](https://user-images.githubusercontent.com/52934128/180646051-aed7efa6-6d15-4853-9962-73186e366f2c.gif) [ ](https://user-images.githubusercontent.com/52934128/180646051-aed7efa6-6d15-4853-9962-73186e366f2c.gif)
> 
> > ## (Highlights) Good Points: 💪
> > 
> > * Good commit messages. ✔️
> > * Followed Github flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> * [x]  Please kindly add a descriptive PR title and description as seen in the demo below:
>   pull_request_desc
>         
>           [
>             
>           
>               pull_request_desc
>             ](https://user-images.githubusercontent.com/52934128/177452738-0610c008-a53a-454e-8aac-635cb4ed812c.gif)
>           
>           
>             
>               
>                 
>               
>               
>                 
>                 
>               
>             
>             [
>               
>                 
>               
>             ](https://user-images.githubusercontent.com/52934128/177452738-0610c008-a53a-454e-8aac-635cb4ed812c.gif)
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
> * Nothing to mention.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag @topeogunleye in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HI! Where can I do this?
Thanks for your help

**Comment by topeogunleye** [comment]
> The changes requested have been solved.

@gdvl777 You haven't added a description to your PR yet.
- [ ] Kindly click on the three dots highlighted with the red arrow to edit your PR description:
image

File: README.md [hunk 4]
```diff
@@ -1 +1,26 @@
...
+
+## Installation
+
+Instructions on how to install and run the project.
+
+## Usage
+
+Instructions on how to use the project.
+
+## Contributing
```
Comment (AlphaNtihinduka) [inline README.md]: As the previous reviewer suggested, In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
Kindly add all sections from the Microverse README and ensure to customize it to be in line with the project repository 👍 . You can refer to this [repository](https://github.com/GhostEsso/Hello-Microverse/tree/htmlcss-branch) to customize your README.
The only optional section from the README file is faq.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @gdvl777 🙋‍♂️
Good job working on implementing the changes required by the previous reviewers. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- [x] You have  reached a great milestone to have the PR description 👍 👏 . However, for this case now, lets focus on how we can make it more professional. I suggest that the PR has a descriptive title(eg: `Hello Microverse`, `Project setup` etc). In addition, the description should be detailed. Kindly consider using some of the project requirements to customize your PR description. This will help other developers or code reviewers to have a great understanding of what the PR is all about ☺️ .

## Optional suggestion
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md (hunk 4 above)
Comment (gdvl777) [reply README.md]: Thank you very much

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @gdvl777 , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Great readme file
✅ ... Good commit messages

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @gdvl777,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] It seems that you are repeating this module because this project was already approved last February, Please make a new project with a new pull request with all project requirements collaboratively with your new team member then submit your project again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**